extension/src/test/suite/experiments/model/filterBy/tree.test.ts

```diff
@@ -95,7 +95,7 @@ suite('Experiments Filter By Tree Test Suite', () => {
               )
             })
             .map(experiment =>
-              experiment.queued
+              experiment.queued || experiment.error
```

Comment:
[F] This points out that the way we don't filter queued will spill over onto error records. You cannot filter by an undefined value which means that "full error" records will be "unfilterable".